cmd/packaging/windows-msi.go

```diff
@@ -102,14 +102,7 @@ var WindowsMsiTask = &packagingTask{
 		},
 	},
 	generateInitFiles: func(packageName, path string) {
-		b := make([]byte, 16)
-		_, err := rand.Read(b)
-		if err != nil {
-			log.Errorf("Failed to generate GUID: %v", err)
-			os.Exit(1)
-		}
-		upgradeCode := strings.ToUpper(fmt.Sprintf("%x-%x-%x-%x-%x", b[0:4], b[4:6], b[6:8], b[8:10], b[10:]))
-		err = ioutil.WriteFile(filepath.Join(path, "upgrade-code.txt"), []byte(fmt.Sprintf("%s\n# This GUID is your upgrade code and ensures that you can properly update your app.\n# Don't change it.", upgradeCode)), 0755)
+		err := ioutil.WriteFile(filepath.Join(path, "upgrade-code.txt"), []byte(fmt.Sprintf("%s\n# This GUID is your upgrade code and ensures that you can properly update your app.\n# Don't change it.", genGUID())), 0755)
```

Comment:
Can we format this a bit nicer? Perhaps put the []byte literal on a separate line with a local varname, or add a comma after `0755` and make it a multi-line function call (each argument on a separate line)